doc/dev/Mgmt-Naming-Conventions.md

```diff
@@ -25,7 +25,7 @@ In addition to adhering to the [General Naming Conventions of .NET](https://lear
 
 **DO** End property or parameter names of type integer that represent intervals or durations with units, for example: `MonitoringInterval` -> `MonitoringIntervalInSeconds`.
 
-**DO** Rename the property or parameter if its name ends with `Interval` or `Duration` and the value is not in ISO 8601 duration format (like P1DT2H59M59S) or ISO 8601 format (like 2.2:59:59.5000000).
+**DO** If a property or parameter name ends with `Interval` or `Duration` and its value is not in ISO 8601 duration format (e.g., P1DT2H59M59S) or ISO 8601 format (e.g., 2.2:59:59.5000000), rename the property or parameter to remove the `Interval` or `Duration` suffix.
```

Comment:
I think `P1DT2H59M59S` is in ISO 8601 format, then `2.2:59:59.5000000` would be a different format, but here we still refer the latter as ISO 8601 format, this is incorrect.

Reply:
`P1DT2H59M59S` is  ISO 8601 duration format, and `2.2:59:59.5000000` is ISO 8601 time format.